code/obj/machinery/manufacturer.dm

```diff
@@ -1922,6 +1922,12 @@
 	blueprint = /datum/manufacture/mechanics/loafer
 
 
+/******************** AI Display Blueprints (should be temporary but we know how that goes in coding) *******************/
```

Comment:
i can try to figure out how to checkout your branch and add them all back in if ya want